payments/verifier.go

```diff
@@ -0,0 +1,41 @@
+package payments
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	"github.com/brave-intl/bat-go/utils/httpsignature"
+)
+
+// the list of payment verifiers, mapping to individuals in payments-ops
+var validVerifiers = map[string]bool{
+	"2b2ddfcfba5045fac57efaf9c6a21e61a0bd7eee3c75e4ad1ee159c7e83cee43": true,
```

Comment:
We should define some rudimentary key management policies for these keys if we haven't already. This is a non-blocking comment for the PR, but I think it's something we should discuss further still.

Most key events (Key Loss, Key Stolen, Key rotation, Key deletion) can all be handled by simply by updating this list of authorized public keys here.

I think we'd want to define a key crypto period as well such as a key should only be used for a maximum of 2 years. We would want to have a place to manage this list so we get alerted to when these keys need to be rotated and who is in possession of them as well.

Just some basic key management policies should suffice here, but I want to double check we've already done it or happy to help establish a basic one for us.

cc @evq

Reply:
I will help create documentation with Kyle to create some key management practices.